core/src/main/java/org/apache/accumulo/core/file/blockfile/cache/BlockCache.java

```diff
@@ -17,10 +17,30 @@
  */
 package org.apache.accumulo.core.file.blockfile.cache;
 
+import org.apache.accumulo.core.conf.AccumuloConfiguration;
+
 /**
  * Block cache interface.
  */
 public interface BlockCache {
+
+  /**
+   * Start the block cache
+   *
+   * @param conf
+   *          Accumulo configuration object
+   * @param maxSize
+   *          maximum size of the on-heap cache
+   * @param blockSize
+   *          size of the default RFile blocks
+   */
+  void start(AccumuloConfiguration conf, long maxSize, long blockSize) throws Exception;
```

Comment:
Also, we could separate the configuration and start logic into separate methods.

Unrelated thought: what about creating a POJO to encapsulate this information. It would prevent drift in the configuration details from breaking blockcache impls (e.g. BlockCacheConfiguration, presently made up of an AccuConf, maxSize, and blockSize).